projects/aca-content/src/lib/ui/variables/variables.scss

```diff
@@ -35,6 +35,10 @@ $action-button-text-color: rgba(33, 35, 40, 0.7);
 $page-layout-header-background-color: #fff;
 $search-chip-icon-color: #757575;
 $disabled-chip-background-color: #f5f5f5;
+$adf-metadata-property-panel-border-color: rgba(0, 0, 0, 0.12);
+$adf-metadata-buttons-background-color: rgba(33, 33, 33, 0.05);
+$adf-snackbar-message-background-color: #ba1b1b;
```

Comment:
same comment like in https://github.com/Alfresco/alfresco-ng2-components/pull/8837/files#r1317260878

Should we really introduce new styles/designs for snackbars instead of using existing snackbars used across application to avoid introducing inconsistency? @DenysVuika @MichalKinas

Reply:
We are removed snackbar theme, and we used existing snackbar design